Sources/MySQL/Bind.swift

```diff
@@ -18,7 +22,7 @@ public final class Bind {
     public typealias CBind = MYSQL_BIND
 
     /// MySQL represents Characters using signed integers.
```

Comment:
Sure, I can fix that. But I'm also curious why it was written in the first place, because I want to make sure I'm not overlooking something. Maybe @tannernelson will know.